apps/src/p5lab/P5Lab.js

```diff
@@ -873,8 +883,9 @@ P5Lab.prototype.reset = function() {
 
 P5Lab.prototype.onPuzzleComplete = function(submit, testResult, message) {
   let msg = this.isSpritelab ? spritelabMsg : gamelabMsg;
-  if (message && msg[message]) {
-    this.message = msg[message]();
+  if (message && this.level.validationStrings[message]) {
+    //this.message = msg[message]();
+    this.message = this.level.validationStrings[message];
```

Comment:
This is an example of getting the string from the validation strings on the level instead of from the i18n!